bootstrap/bootstrap.go

```diff
@@ -243,3 +256,8 @@ func dumpAsJSON(data interface{}) string {
 	}
 	return string(b)
 }
+
+// Generate key file name for migration daemon
```

Comment:
You neeed to begin comment with the name of the item it describes, `GetMigrationDaemonPath ` in this case